packages/cli/src/utils.ts

```diff
@@ -195,12 +195,12 @@ const validateLocalizableResources = (config: CLIConfig) => {
     }
   });
 
-  const descsTooLong = Object.values(config.release.catalog)
+  const descsWrongLength = Object.values(config.release.catalog)
     .map((x) => x.short_description)
-    .filter((desc) => desc.length > 50);
+    .filter((desc) => desc == null || desc.length == 0 || desc.length > 50);
```

Comment:
Do you want to check `== null` here, or do you want to check if the value is defined? In JS, `null` is an assigned value. 

I'd probably make this check `!desc?.length || desc.length > 50`. That'll catch:
- undefined
- length == 0
- length > 50